highlight.io/components/Home/FeatureCarousel/FeatureCarousel.tsx

```diff
@@ -4,25 +4,24 @@ import useEmblaCarousel from 'embla-carousel-react'
 import { StaticImageData } from 'next/image'
 import { useEffect, useState } from 'react'
 import { isMobile } from 'react-device-detect'
-import { AiFillGithub } from 'react-icons/ai'
 import {
 	HiBell,
 	HiChevronDown,
 	HiCloudDownload,
 	HiCode,
 	HiDatabase,
-	HiDesktopComputer,
 	HiDocumentSearch,
+	HiExclamationCircle,
 	HiFilm,
+	HiGlobe,
 	HiLightningBolt,
 	HiPhoneOutgoing,
-	HiPresentationChartLine,
+	HiSparkles,
+	HiTemplate,
 	HiTerminal,
 	HiUserGroup,
 	HiViewBoards,
 } from 'react-icons/hi'
-import selfHosting from '../../../public/images/docker.png'
-import dockerscreenshot from '../../../public/images/dockerscreenshot.png'
```

Comment:
Should we clean up this png since it doesn't appear to be used?

Reply:
Deleted!